### PR TITLE
Project converted from Java to Kotlin

### PR DESCRIPTION
**Pull request by JudahSolomon**
ChannelIem.java,ChanellList.java and Parser.java files have been converted to Kotlin

### CODE REVIEW

File: app/src/main/java/com/enzoftware/androidiptv/m3u/Parser.kt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        val cl = ChannelList()
+        val s = Scanner(playlist).useDelimiter("#")
+        Log.d("HOLINO", s.next().trim { it <= ' ' })
+        require(!s.next().trim { it <= ' ' }.contains("EXTM3U"))
+        while (s.hasNext()) {
+            val line = s.next()
+            val matcher = channel_pattern.matcher(line)
+            require(matcher.find())
+//            val item: ChannelItem = object : ChannelItem()
+//            {
```
Comment (enzoftware) [inline app/src/main/java/com/enzoftware/androidiptv/m3u/Parser.kt:26]: Remove this commented code please 🙏

File: app/src/main/java/com/enzoftware/androidiptv/m3u/ChannelItem.kt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    @JvmField
+    var name: String? = null
+    @JvmField
+    var url: String? = null
+    @JvmField
+    var metadata: HashMap<String, String>? = null
+
+    //    public ChannelItem() {
+    //        metadata = new HashMap<String, String>();
+    //    }
```
Comment (enzoftware) [inline app/src/main/java/com/enzoftware/androidiptv/m3u/ChannelItem.kt:21]: Remove this commented code please 👍

**Review by enzoftware — CHANGES_REQUESTED** [review]
Please take a look on it.

**Comment by Rishavgupta12345** [comment]
ok i am working on it

**Comment by JudahSolomon** [comment]
Thank you very much... i will work on it ASAP

**Pull request closed without merging**